application/common/models/Method.php

```diff
@@ -350,19 +350,30 @@ public static function deleteExpiredUnverifiedMethods()
      * Gets all methods for user specified by $employeeId
      * @param string $employeeId
      * @return String[]
+     * @throws BadRequestHttpException
+     * @throws ServiceException
      */
     public static function getMethods($employeeId)
     {
         $method = new Method;
-        return $method->idBrokerClient->listMethod($employeeId);
+
+        try {
+            return $method->idBrokerClient->listMethod($employeeId);
+        } catch (ServiceException $e) {
+            if ($e->httpStatusCode === 400) {
+                throw new BadRequestHttpException(\Yii::t('app', 'Error locating personnel record'), 1542752270);
```

Comment:
would this actually be a 404?

Reply:
the wording through me off a little, sounded like a message when the user wasn’t found but the idea your describing does sound more like a 500 I think